functions/utils/serve.ts

```diff
@@ -0,0 +1,72 @@
+import { ConnInfo, serve } from "https://deno.land/std@0.167.0/http/server.ts";
+import { MyceliumClient } from "./graphql-client.ts";
+import { makeResp } from "./make-resp.ts";
+
+interface EdgeFunctionHandlerParams<RuntimeDataType = unknown> {
+  myceliumClient: MyceliumClient;
+  request: Request;
+  connInfo: ConnInfo;
```

Comment:
can we avoid abbreviations like this? if I didn't have context I'd be confused with what this means

Reply:
Nope, it's a deno type :p